common/v2/services/LocalCache/constants.ts

```diff
@@ -159,69 +115,38 @@ export const CACHE_INIT_DEV: LocalCache = {
       assets: []
     }
   },
-  nodeOptions: {
-    eth_mycrypto: {
-      name: 'eth_mycrypto',
-      type: 'rpc',
-      service: 'MyCrypto',
-      url: 'https://api.mycryptoapi.com/eth'
-    }
-  },
-  assetOptions: {
-    ETH: {
-      name: 'Ethereum',
-      network: 'ETH',
-      ticker: 'ETH',
-      type: 'base',
-      decimal: 18,
-      contractAddress: null
-    }
-  },
-  contractOptions: {
+  contracts: {
     '17ed6f49-ff23-4bef-a676-69174c266b38': {
       name: 'Athenian: Warrior for Battle',
-      network: 'Ethereum',
+      networkId: 'ETH',
       address: '0x17052d51E954592C1046320c2371AbaB6C73Ef10',
       abi:
         '[{"constant":true,"inputs":[],"name":"name","outputs":[{"name":"","type":"string"}],"payable":false,"type":"function"},{"constant":false,"inputs":[{"name":"_spender","type":"address"},{"name":"_amount","type":"uint256"}],"name":"approve","outputs":[{"name":"success","type":"bool"}],"payable":false,"type":"function"},{"constant":true,"inputs":[],"name":"totalSupply","outputs":[{"name":"totalSupply","type":"uint256"}],"payable":false,"type":"function"},{"constant":false,"inputs":[{"name":"_from","type":"address"},{"name":"_to","type":"address"},{"name":"_amount","type":"uint256"}],"name":"transferFrom","outputs":[{"name":"success","type":"bool"}],"payable":false,"type":"function"},{"constant":true,"inputs":[],"name":"decimals","outputs":[{"name":"","type":"uint8"}],"payable":false,"type":"function"},{"constant":true,"inputs":[{"name":"_owner","type":"address"}],"name":"balanceOf","outputs":[{"name":"balance","type":"uint256"}],"payable":false,"type":"function"},{"constant":true,"inputs":[],"name":"symbol","outputs":[{"name":"","type":"string"}],"payable":false,"type":"function"},{"constant":false,"inputs":[{"name":"_to","type":"address"},{"name":"_amount","type":"uint256"}],"name":"transfer","outputs":[{"name":"success","type":"bool"}],"payable":false,"type":"function"},{"constant":false,"inputs":[{"name":"tokenName","type":"string"},{"name":"tokenSymbol","type":"string"},{"name":"tokenSupply","type":"uint256"}],"name":"SetupToken","outputs":[],"payable":false,"type":"function"},{"constant":true,"inputs":[{"name":"_owner","type":"address"},{"name":"_spender","type":"address"}],"name":"allowance","outputs":[{"name":"remaining","type":"uint256"}],"payable":false,"type":"function"},{"inputs":[{"name":"adr","type":"address"}],"payable":false,"type":"constructor"},{"anonymous":false,"inputs":[{"indexed":true,"name":"_from","type":"address"},{"indexed":true,"name":"_to","type":"address"},{"indexed":false,"name":"_value","type":"uint256"}],"name":"Transfer","type":"event"},{"anonymous":false,"inputs":[{"indexed":true,"name":"_owner","type":"address"},{"indexed":true,"name":"_spender","type":"address"},{"indexed":false,"name":"_value","type":"uint256"}],"name":"Approval","type":"event"}]'
```

Comment:
I might have mentioned this elsewhere but this should be a specific object type or `unknown`